package.json

```
@@ -201,7 +201,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,json,css,md}": [
+    "*.{js,jsx,json,css,md}": [
```

Let's make it the same as our current Prettier extension list (further up in the package.json file)

```suggestion
    "*.{js,jsx,ts,tsx,json,scss,css,yml}": [
```